tests/integration/tests/test_networking.py

```diff
@@ -190,4 +192,6 @@ def test_ipv6_only_on_ipv6_infra(instances: List[harness.Instance]):
         )
 
     # This might take a while
-    util.stubbornly(retries=30, delay_s=20).until(util.ready_nodes(main) == 3)
+    util.stubbornly(retries=config.DEFAULT_WAIT_RETRIES, delay_s=20).until(
```

Comment:
Should we use `DEFAULT_WAIT_DELAY_S` instead of `20`?

Reply:
i don't actually know if this needs 600s to setup or if 150 will do.  Do you know